flashcards/backend/decks/validators.py

```diff
@@ -37,3 +38,43 @@ def validate_file_size(upload):
     """
     if upload.size > settings.MAX_UPLOAD_SIZE:
         raise ValidationError(f"File size is too big. Max size is {settings.MAX_UPLOAD_SIZE} bytes.")
+
+
+def validate_image_file_type(upload):
```

Comment:
In my opinion this validator partly does whats already done in Bartosz "validate_file_type", wouldn't be better to use Bartosz validator, and make this one validate file resolution only.